Tests/RegexBuilderTests/MotivationTests.swift

```diff
@@ -12,15 +12,12 @@
 // FIXME: macOS CI seems to be busted and Linux doesn't have FormatStyle
 // So, we disable this file for now
 
-#if false
-
-import _MatchingEngine
-
 import XCTest
 import _StringProcessing
-
 import RegexBuilder
 
+#if os(macOS)
```

Comment:
Wait, is this all that was needed this whole time? 😅

Reply:
Welp, I guess not. Works on my machine! 🙃